### PR TITLE
Python : How to overcome from EOL Error in Compititve programming... #8008

### LINKED ISSUE #8008: Python : How to overcome from EOL Error in Compititve programming...
### Description

What is EOL and How to overcome from this error in Python.

### Domain

Competitive Programming

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by anjalikumawat2002**
<hr>

## Description 📜

What is EOL and How to overcome from this error in Python.
Fixes #8008 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [x] Competitive Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: Competitive_Programming/Python/EOL Error/EOL Error.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+# How to overcome EOL Error in Competitive Programming
+## EOL 
+
+
+EOL == End Of Line
+
+An EOL error indicatesthat Python interpreter expected a particular character or set of characters to have occurred in specific line of code, but that those characters were not found before the end of the line . This results in Python stopping the program execution and throwing a syntax error .The EOL error signifies that the Interpreter of Python reached the end of the line while scanning the string literal.
```
Comment (ravireddy07) [inline Competitive_Programming/Python/EOL Error/EOL Error.md:7]:
```suggestion
An EOL error indicates that Python interpreter expected a particular character or set of characters to have occurred in specific line of code, but that those characters were not found before the end of the line . This results in Python stopping the program execution and throwing a syntax error .The EOL error signifies that the Interpreter of Python reached the end of the line while scanning the string literal.
```

File: Competitive_Programming/Python/EOL Error/EOL Error.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
+# How to overcome EOL Error in Competitive Programming
+## EOL 
+
+
+EOL == End Of Line
```
Comment (ravireddy07) [inline Competitive_Programming/Python/EOL Error/EOL Error.md:5]:
```suggestion
**EOL == End Of Line**
```

**Review by ravireddy07 — CHANGES_REQUESTED** [review]

**Comment by ravireddy07** [comment]
@ErzaTitania-2001  I haven't approved it yet :man-shrugging:

**Comment by ErzaTitania-2001** [comment]
@ravireddy07 My apologies. I seem to have made a mistake.

**Comment by ravireddy07** [comment]
np, I would have approved. It's ok.  🙌

**Comment by anjalikumawat2002** [comment]
> np, I would have approved. It's ok.  🙌 

Thanks sir

**Pull request merged**